pallets/pool-system/src/tests/mod.rs

```diff
@@ -2439,6 +2439,69 @@ fn create_tranche_token_metadata() {
 	});
 }
 
+#[test]
+fn essence() {
+	new_test_ext().execute_with(|| {
+		let pool_owner = 1_u64;
+		let tranche_input = vec![
+			TrancheInput {
+				tranche_type: TrancheType::Residual,
+				seniority: None,
+				metadata: TrancheMetadata {
+					token_name: BoundedVec::try_from("ResName".as_bytes().to_owned())
+						.expect("String not out of bounds"),
+					token_symbol: BoundedVec::try_from("ResSym".as_bytes().to_owned())
+						.expect("String not out of bounds"),
```

Comment:
NIT. If the test does not check `token_name` or `token_symbol` you can avoid initializing them using directly `BoundedVec::default()` to ease the reading of the test

Reply:
We are checking the tranche metadata in L2498 which is why I did not want to use default values here

Reply:
Ah ok, I didn't see it. Then everything is perfect!